ci/run_envoy_docker.sh

```diff
@@ -22,4 +22,4 @@ docker run --rm -t -i -e http_proxy=${http_proxy} -e https_proxy=${https_proxy}
   -u "${USER}":"${USER_GROUP}" -v "${ENVOY_DOCKER_BUILD_DIR}":/build \
   -v "$PWD":/source -e NUM_CPUS --cap-add SYS_PTRACE "${IMAGE_NAME}":"${IMAGE_ID}" \
   /bin/bash -lc "groupadd --gid $(id -g) -f envoygroup && useradd -o --uid $(id -u) --gid $(id -g) \
-  --no-create-home --home-dir /source envoybuild && su envoybuild -c \"cd source && $*\""
+  --no-create-home --home-dir /source envoybuild && nice -20 su envoybuild -c \"cd source && $*\""
```

Comment:
Can you revert this or open another PR on this if you want to change it?

Reply:
oh rats! that was an accident, will revert